packages/controllers/accounts_modal.coffee

```diff
@@ -1,4 +1,5 @@
 AccountsTemplates.configure
+  showPlaceholders: false
```

Comment:
I thought the placeholders inside the inputs were redundant—they were identical to the input's label.